skrub/_table_vectorizer.py

```diff
@@ -240,8 +198,6 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
         'passthrough' to return the unencoded columns.
         The default transformer is \
             ``OneHotEncoder(handle_unknown="ignore", drop="if_binary")``.
```

Comment:
here we could add a short sentence saying that when using a tree-based model on the resulting features we may want to use a different transformer